packages/notify-client/src/controllers/engine.ts

```diff
@@ -336,6 +337,71 @@ export class NotifyEngine extends INotifyEngine {
     return true;
   };
 
+  public getNotificationHistory: INotifyEngine["getNotificationHistory"] =
+    async ({ topic, limit, startingAfter }) => {
+      this.isInitialized();
+
+      const subscription = this.client.subscriptions.get(topic);
+
+      const identityKey = encodeEd25519Key(
+        await this.client.identityKeys.getIdentity({
+          account: subscription.account,
+        })
+      );
+
+      const issuedAt = Math.round(Date.now() / 1000);
```

Comment:
```suggestion
      const issuedAtInSeconds = Math.round(Date.now() / 1000);
```

When we work with dates, or format them into different periods (hrs, seconds, milliseconds, etc.). It's better to select names that indicate what the value is. This will improve further maintenance and also will allow the reviewer to see possible issues on the next lines if he/she knows what the the time period of the variable is in mind